pkg/model/serializer.go

```diff
@@ -724,10 +724,32 @@ func checkFileAndNotDirectory(path string) error {
 func (d *Dev) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	type devType Dev // Prevent recursion
 	dev := devType(*d)
-	err := unmarshal(&dev)
+
+	type hybridModeInfo struct {
+		Workdir     string            `json:"workdir,omitempty" yaml:"workdir,omitempty"`
+		Selector    Selector          `json:"selector,omitempty" yaml:"selector,omitempty"`
+		Forward     []forward.Forward `json:"forward,omitempty" yaml:"forward,omitempty"`
+		Environment Environment       `json:"environment,omitempty" yaml:"environment,omitempty"`
+		Command     Command           `json:"command,omitempty" yaml:"command,omitempty"`
+		Reverse     []Reverse         `json:"reverse,omitempty" yaml:"reverse,omitempty"`
+		Mode        string            `json:"mode,omitempty" yaml:"mode,omitempty"`
+	}
+	hybridModeDev := &hybridModeInfo{}
+	err := unmarshal(hybridModeDev)
+	if err != nil {
+		if hybridModeDev.Mode == "hybrid" {
+			return err
+		}
+	}
+	err = unmarshal(&dev)
 	if err != nil {
-		return fmt.Errorf("Unmarshal error: '%s'", err)
+		return err
 	}
+
+	if dev.Mode == "hybrid" {
+		dev.Image.Name = "busybox"
+	}
+
 	*d = Dev(dev)
```

Comment:
I can't find it where but If we didn't set the sync folders we were syncthing a path, Can you double check that we are syncthing only one folder and that folder is empty?

Reply:
here https://github.com/okteto/okteto/blob/adrian/add-mode-field-in-dev-section/cmd/up/up.go#L373 we creates a dummy folder to sync